packages/EasyCodingStandard/packages/ChangedFilesDetector/src/FileHashComputer.php

```diff
@@ -3,16 +3,23 @@
 namespace Symplify\EasyCodingStandard\ChangedFilesDetector;
 
 use Nette\Utils\Strings;
-use Symfony\Component\Yaml\Yaml;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+use Symplify\EasyCodingStandard\DependencyInjection\Extension\CheckersExtension;
 
 final class FileHashComputer
 {
     public function compute(string $filePath): string
     {
         if (Strings::endsWith($filePath, '.yml')) {
-            $loadedFileStructure = Yaml::parse(file_get_contents($filePath));
+            $containerBuilder = new ContainerBuilder();
+            $containerBuilder->registerExtension(new CheckersExtension());
 
-            return md5(serialize($loadedFileStructure));
+            $yamlFileLoader = new YamlFileLoader($containerBuilder, new FileLocator(dirname($filePath)));
+            $yamlFileLoader->load($filePath);
+
+            return md5(serialize($containerBuilder));
```

Comment:
don't we wont compute yaml content, instead of container?

Reply:
This covers `imports` sections as well (like previous version for Neon).
Is there any better way to do this?